### DISCUSSION TITLE
Cant open rancher desktop - issue with lima

### DISCUSSION THREAD

**Discussion by PraneethJakkaraju** [rancher-sandbox/rancher-desktop#5242] [Q&A]
<img width="1296" alt="image" src="https://github.com/rancher-sandbox/rancher-desktop/assets/33633524/09b7d16f-025e-448b-ba46-0918aa18c10a">

Mac Ventura 13.5
M2 Pro

**Comment by jandubois** [comment]
There is not enough information in the screenshot to tell what is going on; we will need to be able to see the logs.
